lib/measured/missing_conversion_path.rb

```diff
@@ -0,0 +1,11 @@
+module Measured
```

Comment:
If you're creating new files, can they start with the `# frozen_string_literal: true` comment please?
```suggestion
# frozen_string_literal: true
module Measured
```